jmeter/plugins/jmeter-backendlistener-maestro/README.md

```diff
@@ -0,0 +1,32 @@
+# Maestro Backend Listener for JMeter
+
+Maestro Backend Listener is a JMeter plugins that sends sample results to Maestro API.
+
+## Requirements
+
+  - JDK 1.8
+  - Maven 3.8.x
+
+## Installation
+
+  - Create the JMeter BackendListener Maestro plugin JAR package
+
+    ```bash
+    mvn package
+    ```
+
+  - Put JMeter BackendListener Maestro plugin JAR file into JMeter's lib/ext directory.
+
+## Usage
+The plugin relies on the Jmeter properties that are automatically passed by Maestro Agent. The plugin still could be used separately without maestro if you find a use case for it.
+
+![JMeter backendlistener maestro window](./docs/img/jmeter_backend_listener.png)
+
+ 1. Add JMeter BackendListener Plugin to your test plan.
+ 2. Select `com.farfetch.jmeter.backendlistener.maestro.MaestroBackendClient` Backend Listener Implementation.
+ 3. Set the parameters `maestroUrl`, `maestroAuthToken` and `maestroRunId` accordingly.
+
+## Credits
+
+This plugin contains modified code from the official [JMeter backend listener plugin](
```

Comment:
```suggestion
This plugin contains modified code from the official [Jmeter backend listener plugin](
```

https://github.com/Farfetch/maestro/blob/master/.github/styles/Vocab/Base/accept.txt - we already use it in that way across docs. 
Not sure which one is correct one but would be good to have it in consistent way

Reply:
The correct is JMeter. I want to validate with you if it makes sense to change in other places or here.

Reply:
makes sense to change it to `JMeter` everywhere. You can change all places including `accept.txt` as a separate PR so we can merge it and will keep it the way you have in this PR.

Reply:
created #208 to address the new name convention.